lutris/util/timer.py

```diff
@@ -2,20 +2,30 @@
 
 
 class Timer:
+    """Simple Timer class to time code.
+    You can overload the + operator according to your need by using a new timer_type.
+    """
+
+    def __init__(self, timer_type=None):
+        self.timer_type = timer_type
+        self.finsihed = False
 
     def start_t(self):
-        self.start = datetime.datetime.now()
+        self._start = datetime.datetime.now()
```

Comment:
_start and _end may be referenced before assignment. does start_t expand to start_timer? so Timer.start_timer?